lms/templates/main.html

```diff
@@ -84,31 +94,7 @@
 
   <%block name="headextra"/>
 
-<%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
-    header_file = 'theme-header.html'
-    google_analytics_file = 'theme-google-analytics.html'
-
-    style_overrides_file = None
-
-  else:
-    header_extra_file = microsite.get_template_path('header_extra.html')
-
-    if settings.FEATURES['IS_EDX_DOMAIN'] and not is_microsite():
-        header_file = microsite.get_template_path('navigation-edx.html')
```

Comment:
@saleem-latif @mattdrayer It looks like this PR removed all usages of the file navigation-edx.html, but that file is still in the source tree. Do you see any reason not to remove it? Folks have been changing it since this merged, but seemingly for no benefit.